src/server/common/connectors/notify/notify-token-utils.test.js

```diff
@@ -17,7 +17,8 @@ describe('token-utils', () => {
   })
 
   it('should generate a token with correct payload and options', () => {
-    const iat = Math.round(Date.now() / 1000)
+    const iat = 1
+    jest.spyOn(Math, 'round').mockReturnValue(iat)
```

Comment:
Sorry, I was really imprecise with my original comment.  It'd be better to mock Date.now(), since that's what we want to prove the function does - take the current time, device by 1000, round down and use as appropriate in the JWT token.

Reply:
done